tests/foreman/ui/test_dashboard.py

```diff
@@ -304,20 +523,30 @@ def test_positive_sync_overview_widget(self):
         1.Create a product
         2.Add a repo and sync it
         3.Navigate to Monitor -> Dashboard
-        4.Review the Sync Overview widget
-        for the above sync details
-
-        @Assert: Sync Overview widget is
-        updated with all sync processes
+        4.Review the Sync Overview widget for the above sync details
 
-        @caseautomation: notautomated
+        @Assert: Sync Overview widget is updated with all sync processes
         """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        repo = entities.Repository(
+            url=FAKE_0_YUM_REPO,
+            content_type='yum',
+            product=product,
+        ).create()
+        repo.sync()
+        with Session(self.browser) as session:
+            set_context(session, org=org.name)
+            self.assertEqual(
+                self.dashboard.get_so_product_status(product.name),
+                'Syncing Complete.'
```

Comment:
Is it certain the repo will be synchronized by the time we check the widget? What if it's still in pending state?